definitions/npm/lodash_v4.x.x/flow_v0.63.x/lodash_v4.x.x.js

```diff
@@ -1393,13 +1392,13 @@ declare module "lodash" {
     cond(pairs?: ?NestedArray<Function>): Function;
     conforms(source?: ?Object): Function;
     constant<T>(value: T): () => T;
-    defaultTo<T1: string | boolean | Object, T2>(
+    defaultTo<T1: void | null, T2>(value: T1, defaultValue: T2): T2;
```

Comment:
Move the `null` case to appear first and remove `Object` from the non-null case since it means `any`.